pages/tools/argent-x.mdx

```diff
@@ -0,0 +1,62 @@
+import { Steps } from 'nextra-theme-docs'
+
+# Argent X Overview
+[Argent X](https://www.argent.xyz/argent-x/) is an open-source Starknet wallet utilizing account abstraction for a secure and user-friendly experience. It allows NFT management, interaction with staking and DeFi protocols, and direct crypto purchases within the wallet. 
+
+## Installing Argent X 
+Follow the official Argent X installation instructions: https://www.argent.xyz/learn/how-to-create-an-argent-x-wallet/ 
+
+## Use Argent X in Madara
+Argent X includes the Mainnet, Sepolia, and Goerli networks by default, but establishing interaction with your local Madara chain requires manual configuration. This involves adding a custom network within Argent X's settings.
+
+### Configuring Argent X for Madara
+1. Open the Argent X wallet and navigate to Settings.
+![stack](../../static/argentX/step1.png)
+2. Select "Developer settings" and then "Manage networks".
+![stack](../../static/argentX/step2.png)
+3. Click the "Add network button".
+4. Fill in the following fields:
+    - Network Name: A friendly name for the Madara network.
+    - Chain ID:
+        To retrieve the Madara chain ID:<br></br>
+            1. Open the `madara/blob/main/crates/runtime/src/pallets.rs` file and locate the ChainId variable.
```

Comment:
mixing bullets with numbers is confusing. let's make 1,2,3,4.. as Steps. let's break down 4 into multiple steps then. a separate one for finding the chainId, network name and so on